Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -1168,32 +1200,72 @@ static AffineTransform documentSpaceToPageSpaceTransform(const IntDegrees& pageR
 {
 }
 
-void UnifiedPDFPlugin::setActiveAnnotation(RetainPtr<PDFAnnotation>&& annotation)
+void UnifiedPDFPlugin::setActivePDFPluginAnnotation(RetainPtr<PDFAnnotation>&& annotation)
 {
 #if PLATFORM(MAC)
     if (!supportsForms())
         return;
 
-    if (m_activeAnnotation)
-        m_activeAnnotation->commit();
+    if (m_activePDFPluginAnnotation)
+        m_activePDFPluginAnnotation->commit();
 
     if (annotation) {
 ALLOW_DEPRECATED_DECLARATIONS_BEGIN
         if ([annotation isKindOfClass:getPDFAnnotationTextWidgetClass()] && static_cast<PDFAnnotationTextWidget *>(annotation).isReadOnly) {
-            m_activeAnnotation = nullptr;
+            m_activePDFPluginAnnotation = nullptr;
             return;
         }
 ALLOW_DEPRECATED_DECLARATIONS_END
 
         auto activeAnnotation = PDFPluginAnnotation::create(annotation.get(), this);
-        m_activeAnnotation = activeAnnotation.get();
+        m_activePDFPluginAnnotation = activeAnnotation.get();
         activeAnnotation->attach(m_annotationContainer.get());
     } else
-        m_activeAnnotation = nullptr;
+        m_activePDFPluginAnnotation = nullptr;
     updateLayerHierarchy();
 #endif
 }
 
+void UnifiedPDFPlugin::startAnnotationTracking(RetainPtr<PDFAnnotation>& annotation)
```

Comment:
It would be clearer if the argument were a `RetainPtr<PDFAnnotation>&&` that it's not an out param.

Reply:
Fixed